doc/security/specs/secure_boot/README.md

```diff
@@ -77,13 +77,31 @@ Devices produced before the key is known to be compromised are not protected by
 
 Additionally, each key is restricted to one of three "roles", which determine in which device states the key can be used.
 The roles are:
-* dev (development, only for devices in the `DEV` lifecycle state)
-* test (manufacturing and testing, only for devices in the `TEST_UNLOCK` lifecycle state)
-* prod (production, only for devices in the `PROD` or `PROD_END` lifecycle states)
+* `dev`: development, only for devices in the `DEV` lifecycle state
+* `test`: manufacturing and testing, only for devices in the `TEST_UNLOCK` lifecycle state
+* `prod`: production, only for devices in the `PROD` or `PROD_END` lifecycle states
+The following table summarizes which role can be used in which lifecycle state:
+
+| Key Type | LC_TEST | LC_RMA | LC_DEV | LC_PROD |
+|----------|---------|--------|--------|---------|
+| `dev`    | X       | X      |        |         |
+| `test`   | X       | X      | X      |         |
```

Comment:
It looks like you have the key-types and valid LC states for `dev` and `test` keys mixed up: The `dev` key should be valid for test/rma/dev while the `test` key should be valid only for test/rma.

Reply:
Indeed, I swapped the two, thanks for catching this.